scripts/index.js

```diff
@@ -0,0 +1,13 @@
+const generatePage = require('./generate-page');
+
+function receiveArgs() {
+  const type = process.argv[2];
+  switch (type) {
+    case 'generate-page':
+      generatePage((dir = 'pages'), (args = process.argv));
```

Comment:
The underlying scripts don't care about the first few arguments, so let's strip them out before sending them down, something like:

```
const args = process.argv.slice(3);
switch (type) {
    case 'generate-page':
      generatePage('pages', args);
```